### ISSUE TITLE
how can i add a crf layer into my model?

### ISSUE THREAD

**Issue by SuperMaskv** [deepjavalibrary/djl#1668]
i can't find crf (conditional random field) , plz tell me if there's a crf block in djl. thanks.

**Comment by zachgk** [comment]
There isn't a CRF block in DJL right now. Are you interested in creating it?

**Comment by SuperMaskv** [comment]
I'm sorry that I have no ability to create a CRF block myself, thx for replying. I'm now trying to build my model with pytorch and export using TorchScript. Can djl load, train and predict?

**Comment by zachgk** [comment]
Yeah. If you [export your model with TorchScript](http://docs.djl.ai/docs/pytorch/how_to_convert_your_model_to_torchscript.html), then you can load it with DJL for transfer learning or prediction. You can follow [this example](http://docs.djl.ai/jupyter/load_pytorch_model.html) to load the model.

**Comment by KexinFeng** [comment]
> i can't find crf (conditional random field) , plz tell me if there's a crf block in djl. thanks.

Are you using crf block to train on some dataset, or do you already have a well trained crf model and just want to inference with it?
